mirakuru/output.py

```diff
@@ -45,6 +45,7 @@ def __init__(self, command, banner, **kwargs):
         """
         super(OutputExecutor, self).__init__(command, **kwargs)
         self._banner = re.compile(banner)
+        self.poll_obj = None
```

Comment:
Well that's the `pylint` good practice: not to introduce new instance attributes during lifetime of an object. It helps with unit-tests (you can mock some attribute before a method test) and understanding of a model is easier (you know what entire class consists of without a need of each method studies).